prow/cmd/hook/lgtm.go

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"regexp"
+
+	"k8s.io/test-infra/prow/github"
+)
+
+var (
+	lgtmRe       = regexp.MustCompile(`(?mi)^\/lgtm\r?$`)
+	lgtmCancelRe = regexp.MustCompile(`(?mi)^\/lgtm cancel\r?$`)
+)
+
+// Add or remove LGTM when reviewers comment "/lgtm" or "/lgtm cancel".
+func (ga *GitHubAgent) lgtmComment(ic github.IssueCommentEvent) error {
+	// Only consider open PRs.
+	if ic.Issue.PullRequest == nil || ic.Issue.State != "open" {
+		return nil
+	}
+
+	org := ic.Repo.Owner.Login
+	repo := ic.Repo.Name
+	number := ic.Issue.Number
+
+	// If we create or edit a "/lgtm" comment, add lgtm if necessary.
```

Comment:
Agreed. I'll make it only respond when you create comments. I'm going to remove the behavior where if you delete a `/lgtm` comment it removes LGTM, since I think that's confusing behavior.